front-end/src/app/shared/components/inputs/additional-info-input/additional-info-input.component.ts

```diff
@@ -19,6 +21,8 @@ export class AdditionalInfoInputComponent extends BaseInputComponent implements
     //Add 'implements OnInit' to the class.
 
     this.patchPurposeDescriptionPrefix();
+
+    ValidateUtils.addJsonSchemaValidators(this.form, memoTextSchema, false);
```

Comment:
I noticed that the memo text spec has 
`
            "type": [
                "string",
                "null"
            ],`
  for all of its fields, meaning though they're in the required fields list, we allow null (meaning, it's effectively not required).  If we fix that in the schema, this might stop working because the form will be missing the other fields from the schema.  Not an immediate issue, but we'll want to keep an eye on it

Reply:
nm we are good to go here.  first off, other fields in the text spec are limited to 'string' already.  second it's already been set up on the front end to ignore errors from fields that aren't on the form.  So a field like `rec_type` in the text spec would never impact the transaction form because it's not part of the form.